DemoApp/DemoAppCoordinator.swift

```diff
@@ -131,12 +131,17 @@ private extension DemoAppCoordinator {
             selectedChannel: selectedChannel,
             onLogout: onLogout
         )
-        let channelListNVC = UINavigationController(rootViewController: channelListVC)
         
-        let splitVC = UISplitViewController()
-        splitVC.preferredDisplayMode = .oneBesideSecondary
-        splitVC.viewControllers = [channelListNVC, channelNVC].compactMap { $0 }
-        return splitVC
+        let channelListNVC = UINavigationController(rootViewController: channelListVC)
+        let isIpad = UIDevice.current.userInterfaceIdiom == .pad
+        if isIpad {
+            let splitVC = UISplitViewController()
+            splitVC.preferredDisplayMode = .oneBesideSecondary
+            splitVC.viewControllers = [channelListNVC, channelNVC].compactMap { $0 }
+            return splitVC
+        } else {
+            return channelListNVC
```

Comment:
If we start with `cid`, the app should start with channel screen pushed to the navigation

Reply:
That's already handled, isn't it? `makeChannelListVC` accepts `selectedChannel`